nav2_dijkstra_planner/README.md

```diff
@@ -0,0 +1,25 @@
+# Dijkstra Planner
+The DijkstraPlanner is an implementation of a [Planning module](../doc/requirements/requirements.md) (equivalent to GlobalPlanner in ROS1 [MoveBase](http://wiki.ros.org/move_base)).
+
+The Planning module is responsible for generating a feasible path given start and end robot poses.
```

Comment:
"A planning module implementing the ComputePathToPose interface is responsible..."

Reply:
Thanks, I'll change this too.